docs/en/advanced_guides/tutorials/how_to_use_config_tool_of_pruning.md

```diff
@@ -0,0 +1,199 @@
+# How to Use our Config Tool for Pruning
+
+## How We Get MutableChannelUnits Automatically
+
+Our pruning framework can automatically parse a model and get MutableChannelUnits.
+It makes it easy to prune new models.
+
+The parsing process is placed in ChannelUnitMutator.prepare_from_supernet. We first trace the model and get a graph, then we parse the graph and get MutableChannelUnits.
+
+<p align='center'><img src="../../imgs/pruning/framework-graph.png" width=400 /></p>
+
+## How to Get ChannelUnit Config Template
+
+To make the configuration of ChannelUnit easy, we provide an interface to get the config template: ChannelMutator.config_template(). It returns a config dict. The config\['channel_unit_cfg'\]\['units\] store all parsed MutableChannelUnits.
+
+```python
+def config_template(self,
+                    only_mutable_units=False,
+                    with_unit_init_args=False,
+                    with_channels=False):
```

Comment:
It's better to tell users when to set `only_mutable_units`, `with_unit_init_args` or `with_channels` to True.